accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -251,8 +282,7 @@ initializer expression and is not marked `late` is used before it is definitely
 assigned (see Definite Assignment below).
 
 It is an error if a method, function, getter, or function expression with a
```

Comment:
-> ... if *the body of* a method  ...

(We define the concept for statements above, and methods are not statements).

Reply:
done.